tests/README.md

```diff
@@ -5,7 +5,9 @@
 
 2. Install `kind` https://kind.sigs.k8s.io/
 
-3. Run `bats tests/`
+3. Ensure git submodules have been initialized: `git submodule update --init --recursive`
```

Comment:
this is no longer needed

Reply:
That's for local runs (non CI/CD). 

Added back the submodule commits.